content/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-join.md

```diff
@@ -0,0 +1,55 @@
+---
+title: kubeadm join
+content_type: concept
+weight: 30
+---
+<!-- overview -->
+Este comando inicializa um nó de processamento do Kubernetes e o associa ao
+cluster.
+
+<!-- body -->
+{{< include "generated/kubeadm_join.md" >}}
```

Comment:
Hi @stormqueen1990! The netlify build fails. You need to translate the file generated/kubeadm_join.md.

Reply:
Forgets. It's a draft. Sorry about that.

Reply:
It's now fixed 🎉